std/net/curl.d

```diff
@@ -156,36 +156,26 @@ Distributed under the Boost Software License, Version 1.0.
 */
 module std.net.curl;
 
-import core.thread;
-import etc.c.curl;
-import std.concurrency;
-import std.encoding;
-import std.exception;
-import std.meta;
-import std.range.primitives;
-import std.socket : InternetAddress;
-import std.traits;
-import std.typecons;
-
-import std.internal.cstring;
-
 public import etc.c.curl : CurlOption;
+import std.concurrency : Tid;
+import std.range.primitives;
+import std.encoding : EncodingScheme;
+import std.traits : isSomeChar;
+import std.typecons : Flag, Yes, No, Tuple;
```

Comment:
So when do we get [DIP1005](https://github.com/dlang/DIPs/blob/master/DIPs/DIP1005.md)?